astroquery/cadc/cadctap/tests/DummyConnHandler.py

```diff
@@ -0,0 +1,56 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from astroquery.utils.tap.conn.tests.DummyConnHandler import DummyConnHandler
+
+
+class DummyConnHandlerCadc(DummyConnHandler):
```

Comment:
The documentation for the testing mentions this way of testing so that's why I used it and instead of writing a new way of testing I tried to use most of what was there